### ISSUE TITLE
missing headers in win32/net for network applets

### ISSUE THREAD

**Issue by Blondy314** [rmyorston/busybox-w32#63]
network applets such as ping cannot compile since there are missing files in net folder (for example ip_icmp.h, if_arp.h etc).

Is there a soltuin for this? where can I get the required headers

**Comment by rmyorston** [comment]
Yeah, if something isn't enabled in the default mingw configuration it probably won't compile. Or if it compiles it probably won't work.

If mingw doesn't provide a required include file you'll need to create a suitable replacement. This can be based on the Linux equivalent but it only really has to contain the stuff your applet needs. It may also be necessary to implement any functions that aren't provided in the WIN32 API.

I haven't bothered with the ```ping``` applet because Windows has its own ```ping``` binary.

**Comment by Blondy314** [comment]
I agree the ping Window already has ping but it is useful to have all the applets inside one application so you don't need to run external applications. The main reason I want it is so I can play with the flags and IP options in the packets and so on.

Has anyone tried (and succeeded) to compile the network applets?
Furthermore, did anyone compile the busybox in Visual Studio without mingw ?

**Comment by rmyorston** [comment]
A handful of network applets are enabled in the default configuration but I myself haven't tried to build most of the rest. I can't speak for anyone else.

I do all of my development work on Linux using mingw. I haven't heard of anyone using Visual Studio and I suspect it wouldn't work.
